srcCpp03/perftestThreadPriorities.cxx

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) 2005-2018  Copyright, Real-Time Innovations, Inc. All rights reserved.
+ * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
+ */
+
+#include "perftestThreadPriorities.h"
+
+PerftestThreadPriorities::PerftestThreadPriorities()
+        : main(0), receive(0), dbAndEvent(0), isSet(false)
+{
+    #ifdef RTI_WIN32
+        defaultPriorities['h'] = THREAD_PRIORITY_TIME_CRITICAL;
+        defaultPriorities['n'] = THREAD_PRIORITY_NORMAL;
+        defaultPriorities['l'] = THREAD_PRIORITY_IDLE;
+    #elif RTI_UNIX
+        defaultPriorities['h'] = sched_get_priority_max(SCHED_FIFO);
+        defaultPriorities['n'] = (sched_get_priority_max(SCHED_FIFO)
+                + sched_get_priority_min(SCHED_FIFO)) / 2;
+        defaultPriorities['l'] = sched_get_priority_min(SCHED_FIFO);
+    #endif
```

Comment:
Add another #else and print a comment saying:

PerftestThreadPriorities is only implemented for Windows and Unix Operative Systems.

Reply:
But then, in any execution, even if you are using thread or not, that message will appear if the system does not support it.
Right now, if we don't support thread priorities on a system, the message will appear only if the command line is provided.

Reply:
got it, works for me.